enterprise/frontend/src/metabase-enterprise/auth/components/SettingsSAMLForm/SettingsSAMLForm.jsx

```diff
@@ -82,12 +89,13 @@ const SettingsSAMLForm = ({ elements = [], settingValues = {}, onSubmit }) => {
           <h3 className="mb0">{t`Configure your identity provider (IdP)`}</h3>
           <p className="mb4 mt1 text-medium">{t`Your identity provider will need the following info about Metabase.`}</p>
 
-          <div className="Form-field">
-            <div className="Form-label">{t`URL the IdP should redirect back to`}</div>
-            <div className="pb1">{t`This is called the Single Sign On URL in Okta, the Application Callback URL in Auth0,
-                                  and the ACS (Consumer) URL in OneLogin. `}</div>
-            <CopyWidget value={getAcsCustomerUrl()} />
-          </div>
+          <FormTextInput
+            name={FAKE_ACS_URL_KEY}
+            description={t`This is called the Single Sign On URL in Okta, the Application Callback URL in Auth0, and the ACS (Consumer) URL in OneLogin. `}
+            hasCopyButton={true}
+            readOnly={true}
```

Comment:
we usually just use the react shorthand:
```
hasCopyButton
readOnly
```
which implies true

Reply:
https://github.com/metabase/metabase/pull/35731/commits/0ce87ad2b29c3e8b4f50f690f1e7f4cafcf3536f